pkg/sdk/external_tables_impl.go

```diff
@@ -0,0 +1,99 @@
+package sdk
+
+import "context"
+
+var (
+	_ ExternalTables                = (*externalTables)(nil)
```

Comment:
You have this twice. PLease remove it from other file (I mean the ExternalTables one)